src/user/models/kyc/kyc.service.ts

```diff
@@ -175,13 +183,44 @@ export class KycService {
     const dataComplete = this.isDataComplete(user);
     if (!dataComplete) throw new BadRequestException('Ident data incomplete');
 
+    const users = await this.userDataService.getUsersByInformation(user);
+    if (users) {
+      const completedUser = users.find((data) => KycCompleted(data.kycStatus));
+      const shouldSendLinkEmail = users.length > 1;
+      if (shouldSendLinkEmail && completedUser) {
```

Comment:
`shouldSendLinkEmail` is always true, if `completedUser` is not null => can be removed

Reply:
yes, removed